src/components/Posts/PostsMaker/PostMakerContainer/PostPermissionsSection.tsx

```diff
@@ -76,16 +77,16 @@ export default function PostPermissionsSection({
         <div className="mt-3 px-3">
           <div className="flex items-center gap-3">
             <p
-              className="text-color-primary 
+              className="text-color-1 
                 text-base font-medium">
               Only to people who hold at least
             </p>
             <input
               type="number"
               name="tokenrequirement"
               id="tokenrequirement"
-              className="border border-gray-300 rounded-md block 
-              focus:ring-0 text-color-primary text-sm  px-3 text-left 
+              className="text-field-input border 
+               rounded-xl block text-color-primary text-sm px-3 text-left 
               w-28 h-8"
```

Comment:
Should the text-color-primary be replaced by text-color-1 here too?

As for the w-28 h-8, would it be possible to replace by fractional widths, like w-1/12....I think when possible we should use sizes that are relative to the screen size to improve responsiveness.

Reply:
yep